### ISSUE TITLE
Cert is not issues

### ISSUE THREAD

**Issue by Rahulsharma0810** [math-nao/certs#47]
```
Version: 1.1.10
wait few seconds in case ingress rule is deployed at the same as it is in demo
Before script
Info: Initialize environment...
Info: No cmd args found
Info: Generate certs for dns: dns_gd, is_staging: false, is_debug: false, args: , cmd to use: acme.sh --register-account -m me@rvsharma.com, name: ca-kubedash-rvsharma-cert-secret, namespace: kubernetes-dashboard, domains: kubedash.labs.rvsharma.com
Info: Loading conf from secret...
Info: Invalid status code found: 404, configuration not loaded
[Tue Aug 17 10:13:08 UTC 2021] Create account key ok.
[Tue Aug 17 10:13:08 UTC 2021] No EAB credentials found for ZeroSSL, let's get one
[Tue Aug 17 10:13:09 UTC 2021] Registering account: https://acme.zerossl.com/v2/DV90
[Tue Aug 17 10:13:13 UTC 2021] Registered
[Tue Aug 17 10:13:13 UTC 2021] ACCOUNT_THUMBPRINT='lwwxDxUNfq2WUUxKGnVNSPdGeCAsIx7okXGgGzLHLJlxk'
Info: acme.sh return code: 0
Info: No certificate change, nothing to do
Exiting...
After script
```

On every cron pod it's returning the same output, with any ingress.

**Comment by math-nao** [comment]
It seems related to zerossl acme.sh procedure to generate certificates. Try to add your command to register account and your command to issue certificate to `acme.kubernetes.io/cmd-to-use`.

**Comment by Rahulsharma0810** [comment]
```annotations:
    kubernetes.io/ingress.class: nginx
    # nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
    acme.kubernetes.io/enable: "true"
    acme.kubernetes.io/dns: "dns_gd"
    acme.kubernetes.io/cmd-to-use: "acme.sh --register-account -m me@rvsharma.com"
    nginx.ingress.kubernetes.io/rewrite-target: /```

**Comment by math-nao** [comment]
Try this workaround:   `acme.kubernetes.io/cmd-to-use: "acme.sh --register-account -m me@rvsharma.com; acme.sh --server zerossl --issue  -d  example.com --dns dns_cf"`

**Comment by TomKeyte** [comment]
I saw this the other day, it looked related to acme.sh version 3
I updated the Dockerfile to use `neilpang/acme.sh:2.9.0` and it worked fine

**Comment by math-nao** [comment]
Use the following ingress annotation: `acme.kubernetes.io/add-args: "--server letsencrypt"`

**Comment by math-nao** [comment]
@Rahulsharma0810 any news? did you try with previous workaround or with latest version (without workaround)?

**Comment by Rahulsharma0810** [comment]
I actually migrated from GoDaddy DNS to GCP DNS and used cert manager.

**Comment by math-nao** [comment]
Ok, thank you for you reply. I will close this issue because issue is solved on `1.1.11` version and a workaround has been given for previous version.
